x/staking/client/cli/query.go

```diff
@@ -17,7 +17,11 @@ func GetCmdQueryValidator(storeName string, cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
 		Use:   "validator [operator-addr]",
 		Short: "Query a validator",
-		Args:  cobra.ExactArgs(1),
+		Long: `Query details about an individual validator:
+
```

Comment:
Brilliant - I feel we should definitely expand all commands usage screens following this example.

Reply:
I'm in process of doing that.